docs/website/content/docs/v0.1/Getting Started/resources.md

```diff
@@ -0,0 +1,119 @@
+---
+description: ""
+weight: 4
+---
+
+# Resources
+
+Sidero, the Talos bootstrap/controlplane providers, and Cluster API each provide several custom resources (CRDs) to Kubernetes.
+These CRDs are crucial to understanding the connections between each provider and in troubleshooting problems.
+It may also help to look at the [cluster template](https://github.com/talos-systems/sidero/blob/master/templates/cluster-template.yaml) to get an idea of the relationships between these.
+
+---
+
+## Cluster API (CAPI)
+
+It's worth defining the most basic resources that CAPI provides first, as they are related to several subsequent resources below.
+
+### Cluster
+
+Cluster is the highest level CAPI resource.
+It allows users to specify things like network layout of the cluster, as well as contains references to the infrastructure and control plane resources that will be used to create the cluster.
+
+### Machines
+
+Machine represents an infrastructure component hosting a Kubernetes node.
+Allows for specification of things like Kubernetes version, as well as contains reference to the infrastructure resource that relates to this machine.
+
+### MachineDeployments
+
+MachineDeployments are similar to a Deployment and their relationship to Pods in Kubernetes primitives. 
+A MachineDeployment allows for specification of a number of Machine replicas with a given specification.
```

Comment:
Should we wrap types in back ticks to make them code? e.g. `MachineDeployment`.

Reply:
I can do that. hang tight.

Reply:
okay, updated.